ml-agents/mlagents/trainers/trainer.py

```diff
@@ -98,7 +109,10 @@ def get_action(self, curr_info: BrainInfo) -> ActionInfo:
         :param curr_info: Current BrainInfo.
         :return: The ActionInfo given by the policy given the BrainInfo.
         """
-        return self.policy.get_action(curr_info)
+        self.trainer_metrics.start_experience_collection_timer()
```

Comment:
In BC trainer that inherits from trainer but no trainer_metrics is used, will this cause problems ?

Reply:
No because `trainer_metrics` is of type `TrainerMetrics` which is instantiated in the constructor for `Trainer`, the constructor is called in `BCTrainer` so the object will exist and the method can be called. We will not write the data because it will be gibberish.